main.c

```diff
@@ -67,10 +27,20 @@ void testPairToNumber(
     assert(pairNumber == expectedPairNumber);
 }
 
+void colorMapperManual()
+{
+ 	 ColorPair colorPair;
+     for(int pairNumber=1; pairNumber<26; pairNumber++)
```

Comment:
Do not hardcode numbers like 26. They would be hard to maintain when adding or removing colors